tox.ini

```diff
@@ -4,12 +4,21 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py37,py38,py39,py310
+envlist = 
+    py37
+    py38
+    py39
+    py310
+isolated_build = True
 
 [testenv]
-deps =
-    pytest
-    coverage
-    typing_extensions
+extras =
+    tox_only
 commands =
     coverage run --source=screenpy -m pytest {posargs}
+
+[tox:.package]
```

Comment:
What does this section do?

Reply:
TBH, I'm not exactly sure why this is present.  Found it in the documentation https://tox.wiki/en/latest/example/package.html#poetry

Reply:
Reading that comment, it seems like this section is unnecessary unless we expect people to be installing `tox` under Python2. We can probably omit it.

Reply:
I agree. I'll remove it.